### PR TITLE
Data Science Team: Business establishments  (land use) and jobs (100%)

### PR DESCRIPTION
**Pull request by HarshDW09**
Hi  @VaishaliThakoor &  @kitkatch  ,

This is the final pull request 

•	Cleaned the dataset to ensure all necessary columns were present and handled missing values.
•	Standardized and scaled the features for better model performance.
•	Applied XGBoost regression to predict job density.
•	Used GridSearchCV to optimize hyperparameters, resulting in improved model performance.
•	Achieved high predictive accuracy with an R-squared score of 0.85.
•	Identified key features impacting job density through feature importance analysis.
•	Conducted spatial analysis to identify clusters of high job density.
•	Provided actionable insights and recommendations for urban planners based on the model’s predictions.
•	Provided suggestion to  the planner.

### CODE REVIEW

**Review by VaishaliThakoor — CHANGES_REQUESTED** [review]
Hey @HarshDW09 great job on the usecase! The graphs are very detailed and explicit and the explanation for each part is very good as well!
I would just request a minor change in your map - in the legend you just need to put “Hotspot” once as the red dot instead of putting Hostpot 1,2,3,4 in the legend since your map only shows hotspots as the red dots and when you hover over them it already shows which hotspot it is! 
Otherwise everything else is great and good to go!

**Comment by HarshDW09** [comment]
Hi @VaishaliThakoor & @kitkatch  ,
Thanks for the review 
I've made the changes you requested. Specifically, I updated the legend to display "Hotspot" as a single entry with a red dot, instead of listing Hotspot 1, 2, 3, 4, and 5 individually. 


Screenshot (370)

**Review by VaishaliThakoor — APPROVED** [review]
Great! And amazing work again @HarshDW09
